src/main/java/org/wildfly/security/sasl/gssapi/AbstractGssapiMechanism.java

```diff
@@ -172,7 +172,7 @@ public Object getNegotiatedProperty(String propName) {
                 return Integer.toString(maxBuffer);
         }
 
-        return null;
+        return super.getNegotiatedProperty(propName);
```

Comment:
The superclass only repeats the assertComplete() call.  I don't think this PR adds any value.

Reply:
I agree that the current implementation doesn’t change the behaviour but from a design point of view it unifies with the rest of the subclasses. Theoretically the behavior will change with future superclass changes.